torch_xla/csrc/ops/einsum_backward.cpp

```diff
@@ -0,0 +1,134 @@
+#include "torch_xla/csrc/ops/einsum_backward.h"
+
+#include "tensorflow/compiler/xla/client/lib/matrix.h"
+#include "torch_xla/csrc/helpers.h"
+#include "torch_xla/csrc/lowering_context.h"
+#include "torch_xla/csrc/ops/einsum.h"
+#include "torch_xla/csrc/ops/einsum_utilities.h"
+#include "torch_xla/csrc/ops/infer_output_shape.h"
+#include "torch_xla/csrc/ops/xla_ops.h"
+
+namespace torch_xla {
+namespace {
+
+std::vector<torch::lazy::Value> GetOperandList(
+    c10::ArrayRef<torch::lazy::Value> operands,
+    const torch::lazy::Value& grad_output) {
+  std::vector<torch::lazy::Value> operand_list(operands.begin(),
+                                               operands.end());
+  operand_list.insert(operand_list.begin(), grad_output);
+  return operand_list;
+}
+
+std::vector<xla::XlaOp> BuildEinsumBackward(const xla::XlaOp& grad_output,
+                                            absl::Span<const xla::XlaOp> inputs,
+                                            const std::string& equation) {
+  std::vector<xla::XlaOp> ops;
```

Comment:
nit, maybe `results`? `ops` sounds like a input to something :D.

Reply:
Done